feat : added activity item card

hibabdelk opened the pull request.

hibabdelk committed:
feat : added activity item card

osamaakb commented on src/Components/ActivitiesItem/ActivitiesItem.js:
```diff
@@ -0,0 +1,81 @@
+
+const { Title } = Typography;
+
+export default function ActivitiesItem({
```
can you send these as an object? you can do that by adding extra parenthesis

osamaakb commented on src/Components/ActivitiesItem/ActivitiesItem.js:
```diff
@@ -0,0 +1,81 @@
+  };
+
+  return (
+    <Card style={{ height: 200, width: 600 }}>
```
make this hoverable

osamaakb commented on src/Components/ActivitiesItem/ActivitiesItem.js:
```diff
@@ -0,0 +1,81 @@
+                  />
+                ) : (
+                  <HeartFilled
+                    style={{ fontSize: '30px', color: 'red' }}
```
let's put these in separate css file

osamaakb reviewed (changes requested):
Good job Hiba, please read the comments

hibabdelk committed:
Refactor : fixed the activity item card
body : added a badge in case the tags are more than 4 to show 10+ for example and made the card hoverable
